pmdk.spec

```
@@ -390,6 +387,9 @@ cp utils/pmdk.magic %{buildroot}%{_datadir}/pmdk/
 # PMDK tends to document multiple functions in a single groff file which
 # translates into multiple copies of one file.
 %fdupes -s %{buildroot}%{_mandir}
+# Remove all libpmem2 related files as they are excluded from installation
+find %{buildroot} -name "libpmem2*" -type f -exec rm -f {} \;
```

FWIW, it's much more efficient to do: 

```suggestion
find %{buildroot} -name "libpmem2*" -type f -print0 | xargs -r0 rm -f
```
As it greatly reduces the number of `rm -f` commands executed.

But ultimately that `find` does not seem to be working per the latest run.

Can you remind me why we are trying to exclude pmem2 from the RPM?

Would it be better to have a build option that skips pmem2 in such a case?

---

Let's make this change on the PMDK side with a new flag (this will be included in 2.1.1 release).

`libpmem2` library is not used at all by DAOS.